### ISSUE TITLE
Croatia: "HSP AS" and "HSP dr. Ante Starčević" & "Labourists" and "Croatian Labourists – Labour Party"

### ISSUE THREAD

**Issue by marfi** [everypolitician/everypolitician-data#7744]
In https://github.com/everypolitician/everypolitician-data/blob/master/data/Croatia/Sabor/term-8.csv we encounter "HSP AS" under "party".

In https://github.com/everypolitician/everypolitician-data/blob/master/data/Croatia/Sabor/term-7.csv there is: "HSP dr. Ante Starčević" in the same field. In both datasets however there's no mention of the terms simultaneously.

I am pretty confident that "HSP AS" is the abbreviation of the initials of Ante Starcevic in "HSP dr. Ante Starčević". I will investigate this further. And report back.

I think the same applies to "Labourists" and "Croatian Labourists – Labour Party".

**Comment by tmtmtmtm** [comment]
Yep — I reached the same conclusion when I looked at this a couple of days ago: https://github.com/everypolitician/everypolitician-data/blob/8f7a98e103e7177073e0350c1f3be71b98783188/data/Croatia/Sabor/sources/manual/group_wikidata.csv#L12-L13

Having this actually _become_ the same Group in the source is non-trivial though. If it's important to you, we could manually set these to the same thing in the scraper, but I'd rather fix it in a more consistent manner everywhere. (It's complicated by the fact that parties and factions legitimately change name, and whilst you want to be able to track that they're still the same entity, you want to be able to use the name they were known by at a given point, rather than just always use the most recent one)

**Comment by marfi** [comment]
It is not crucial for me. I agree that it needs to be fixed more globally.
